javascript/HW4/src/app/article-list/article-details/article-details.component.html

```diff
@@ -1,12 +1,10 @@
 <div>
-    <p>
-        Стаття з рубрики: {{heading}}
-    </p>
+    <p>Стаття з рубрики: {{heading}}</p>
     <h4>
-        <a (click)="changeColor()"><font color="{{color}}">{{ article.title }}</font></a>
```

Comment:
Do not use the font tag, since it was deprecated in HTML 4.01.
http://w3schools.sinsixx.com/tags/html5_font.asp.htm